### ISSUE TITLE
Additional Discovery Search cohort for Demo

### ISSUE THREAD

**Issue by ljdursi** [CINECA-project/wp1_project_management#14]

**Comment by ljdursi** [comment]
- On hold while implementation is built (e.g. blocked on #15)
- Federation will be more complicated; having 2 Search cohorts for March may not be significantly more useful than 1.  Will need for October

**Comment by ljdursi** [comment]
Moving this to "not yet"; will aim for MTR not March Demo
